{"uid":"2fe19397062a2b88","name":"4. should display \"An unexpected error occurred\" for a network failure","fullName":"tests/Login.test.tsx#The Login Page Use Cases 4. should display \"An unexpected error occurred\" for a network failure","historyId":"acd02362ef57b79210e72d2f333e11b2:d41d8cd98f00b204e9800998ecf8427e","time":{"start":1761994096615,"stop":1761994096828,"duration":213},"status":"passed","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":false,"beforeStages":[],"afterStages":[],"labels":[{"name":"framework","value":"vitest"},{"name":"language","value":"javascript"},{"name":"parentSuite","value":"The Login Page Use Cases"},{"name":"host","value":"runnervmf2e7y"},{"name":"thread","value":"vitest-worker-1"},{"name":"package","value":"tests.Login.test.tsx"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"c8bddf80399d0493","status":"passed","time":{"start":1761991095697,"stop":1761991095834,"duration":137}}],"categories":[],"tags":[]},"source":"2fe19397062a2b88.json","parameterValues":[]}